docs/quickstart.rst

```diff
@@ -8,51 +8,140 @@ up pg_auto_failover to replicate data between them. We’ll simulate failure in
 the primary node and see how the system smoothly switches (fails over)
 to the secondary.
 
-For simplicity we’ll run all the pieces on a single machine, but in
-production there would be three independent machines involved for each
-managed PostgreSQL service. One machine for the primary, another for
-the secondary, and the last as a monitor which watches the other nodes’
-health, manages global state, and assigns nodes their roles.
+For illustration, we'll run our databases on virtual machines in the Azure
+platform, but the techniques here are relevant to any cloud provider or
+on-premise network. We'll use three virtual machines: a primary database, a
+secondary database, and a monitor which watches the other nodes’ health,
+manages global state, and assigns nodes their roles.
+
+.. _quickstart_network:
+
+Create virtual network
+----------------------
+
+Our database machines need to talk to each other and to the monitor node, so
+let's create a virtual network.
+
+.. code-block:: bash
+
+   az group create \
+       --name ha-demo \
```

Comment:
we have ha-demo repeated everywhere, I used following

```bash
export RESOURCE_GROUP=mt_citusha_1
export VNET_NAME=${RESOURCE_GROUP}_vnet
export SUBNET_NAME=${RESOURCE_GROUP}_subnet
export NSG_NAME=${RESOURCE_GROUP}_nsg
export USERNAME=citus
```

This helped me a lot when using a different resource group than `ha-demo`

Reply:
`$USERNAME` was a poor choice it was mixed up with system username, now switched to `$ADMIN_USER_NAME`